wazo_confd/plugins/event_handlers/service.py

```diff
@@ -0,0 +1,191 @@
+            ],
+            'auth_section_options': [],
+            'endpoint_section_options': [
+                ['rtp_timeout', '7200'],
+                ['allow_transfer', 'yes'],
+                ['use_ptime', 'yes'],
+                ['callerid', 'wazo'],
+                ['direct_media', 'no'],
+                ['dtmf_mode', 'rfc4733'],
+                ['language', 'en_US'],
+                ['inband_progress', 'no'],
+                ['rtp_timeout_hold', '0'],
+                ['timers_sess_expires', '600'],
+                ['timers_min_se', '90'],
+                ['trust_id_inbound', 'no'],
+                ['allow_subscribe', 'yes'],
+            ],
+            'registration_section_options': [],
+            'registration_outbound_auth_section_options': [],
+            'identify_section_options': [],
+            'outbound_auth_section_options': [],
+            'templates': [],
+        }
+        global_template = self.create_or_merge_sip_template(
+            global_config, tenant.global_sip_template_uuid,
+        )
+
+        webrtc_config = {
+            'label': 'webrtc',
+            'template': True,
+            'tenant_uuid': tenant.uuid,
+            'transport': transport_wss,
+            'asterisk_id': None,
+            'aor_section_options': [],
+            'auth_section_options': [],
+            'endpoint_section_options': [
+                ['webrtc', 'yes'],
+                ['dtls_auto_generate_cert', 'yes'],
+                ['allow', '!all,opus,g722,alaw,ulaw,vp9,vp8,h264'],
+            ],
+            'registration_section_options': [],
+            'registration_outbound_auth_section_options': [],
+            'identify_section_options': [],
+            'outbound_auth_section_options': [],
+            'templates': [global_template],
+        }
+        webrtc_template = self.create_or_merge_sip_template(
+            webrtc_config, tenant.webrtc_sip_template_uuid,
+        )
+
+        global_trunk_config = {
+            'label': 'global_trunk',
+            'template': True,
+            'tenant_uuid': tenant.uuid,
+            'transport': None,
+            'asterisk_id': None,
+            'aor_section_options': [],
+            'auth_section_options': [],
+            'endpoint_section_options': [],
+            'registration_section_options': [
+                ['forbidden_retry_interval', '30'],
+                ['retry_interval', '20'],
+                ['max_retries', '10000'],
+                ['auth_rejection_permanent', 'no'],
+                ['fatal_retry_interval', '30'],
+            ],
+            'registration_outbound_auth_section_options': [],
+            'identify_section_options': [],
+            'outbound_auth_section_options': [],
+            'templates': [global_template],
+        }
+        global_trunk_template = self.create_or_merge_sip_template(
+            global_trunk_config, tenant.global_trunk_sip_template_uuid,
+        )
+
+        twilio_trunk_config = {
```

Comment:
How can we removed this hardcoded template?

Reply:
:man_shrugging: It's up to you to tell me, it was already hardcoded before (in wazo-confgend)